src/test/java/com/sap/oss/phosphor/fosstars/tool/github/MergedJsonReporterTest.java

```diff
@@ -0,0 +1,43 @@
+package com.sap.oss.phosphor.fosstars.tool.github;
+
+import static com.sap.oss.phosphor.fosstars.tool.github.AbstractReporter.LIST_OF_GITHUB_PROJECTS_TYPE;
+import static java.util.Arrays.asList;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+import com.sap.oss.phosphor.fosstars.model.subject.oss.GitHubProject;
+import com.sap.oss.phosphor.fosstars.util.Json;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.apache.commons.io.FileUtils;
+import org.junit.Test;
+
+public class MergedJsonReporterTest {
+
+  @Test
+  public void testLoadAndSave() throws IOException {
+    Path path = Files.createTempFile(MergedJsonReporterTest.class.getName(), "test");
```

Comment:
Do we not introduce `GitHubOrganization` for the serialization and deserialization test here?